common/source/docs/common-imu-notch-filtering.rst

```diff
@@ -6,37 +6,48 @@
+#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 3. ESC Telemetry based, where the ESC provides motor RPM information which is used to set the center frequency.
+#. :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 4. In-Flight FFT, where a running FFT is done in flight to determine the primary noise frequency and adjust the notch's center frequency to match.
+
 -----
 Setup
 -----
 
 Pre-Flight Setup
 ================
 
-ArduPilot allows you to select the mechanism used for controlling the harmonic notch frequency. By default this mechanism is :ref:`throttle-based<common-imu-notch-filtering-throttle-based-setup>` since that will work on all Copters/QuadPlanes. However, for Copters with an rpm sensor or :ref:`BLHeli ESC telemetry support<common-dshot-blheli32-telemetry>`, setup can be radically simpler:
+First you must select the mechanism used for controlling the harmonic notch frequency. By default (:ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1) this mechanism is :ref:`throttle-based<common-imu-notch-filtering-throttle-based-setup>` since that will work on all Copters/QuadPlanes (. And therefore, a flight log analysis will be required to correlate the hover throttle position to the noise sources frequency.
 
-- Set :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 2 to use the RPM sensor to set the harmonic notch frequency. This is primarily used in :ref:`Helicopters<common-imu-notch-filtering-helicopter-setup>`
+However, for Copters with an rpm sensor or :ref:`BLHeli ESC telemetry support<common-dshot-blheli32-telemetry>`, setup can be radically simpler by using direct RPM sensing, which will not require log analysis to determine center frequency:
+
+- Set :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 2 to use an RPM sensor to set the harmonic notch frequency. This is primarily used in :ref:`Helicopters<common-imu-notch-filtering-helicopter-setup>`
 - Set :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 3 to use BLHeli ESC telemetry support to set the harmonic notch frequency. This requires that your ESCs are configured correctly to support BLHeli telemetry via :ref:`a serial port<common-dshot-blheli32-telemetry>`
 - Set :ref:`INS_HNTCH_REF <INS_HNTCH_REF>` = 1 to set the harmonic notch reference value, which for ESC telemetry generally means no scaling
 
 .. _common-imu-notch-filtering-throttle-based-setup:
 
+--------------------
 Throttle-based Setup
-====================
-
-If you do not have ESC telemetry then throttle-based setup is generally the way to go.
+--------------------
 
-- Set :ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1 to use a throttle-based estimation to set the harmonic notch frequency.
+If you do not have ESC telemetry or an RPM sensor, then throttle-based (:ref:`INS_HNTCH_MODE <INS_HNTCH_MODE>` = 1 ) center frequency control is best.
```

Comment:
Not sure about best, FFT is generally better and easier to set up if you have the CPU cycles

Reply:
will change appropriately